.github/workflows/build.yml

```diff
@@ -29,6 +29,12 @@ jobs:
     - name: Build with Gradle Wrapper
       run: ./gradlew shadowJar
 
+    - name: Upload build
+      uses: actions/upload-artifact@v4.3.3
+      with:
+        name: 'DeluxeMenus build PR-${{ github.event.number }} (commit ${{ github.sha }})'
```

Comment:
Can we set the environment variable BUILD_NUMBER to be the PR and the commit hash instead? I haven't tested this but it seems it might be possible  in the `Build with Gradle Wrapper` step like this: `run: BUILD_NUMBER="PR-${{ github.event.number }}-${{ github.sha }}" ./gradlew shadowJar`